src/components/views/RateWhites.js

```diff
@@ -27,14 +27,21 @@ const RateWhites = () => {
         async function fetchVote() {
             try { //Getting a "Vote" which is a game (BlackCard with all it's played WhiteCards)
                 const response = await api.get(`users/${userId}/games/vote`);
-
                 setBlackCard(response.data.blackCard)
                 setPlays(response.data.plays)
                 setCurrentPLay(response.data.plays.shift()) //This removes the first item of the array, which is the first play
             }
             catch(error){
-                console.error("Details", error);
-                alert("Problem with getting the Game");
+                // when no active game left or past game with open plays
+                if (error.response.status === 404) {
+                    // message: when no active game left or past game with open plays
+                    setBlackCard(null);
+                    setCurrentPLay(null);
+                    console.error("Error 404: ", error.response.data.message)
```

Comment:
It's not really an error though

Reply:
status is not found, when trying to get a game with plays.
for me it is clearer als the answer "" when no game to vote on is found
but we can change it
check on response.fata==null is not working